Makefile

```diff
@@ -325,7 +325,9 @@ ifndef COURIER_PACKAGE_VERSION
 endif
 
 .PHONY: csv
-csv: check-package-version operator-sdk
+csv: deploy/olm-catalog/compliance-operator/$(COURIER_PACKAGE_VERSION) check-package-version operator-sdk ## Generate the CSV and packaging for the specific version (NOTE: Gotta specify the version with the COURIER_PACKAGE_VERSION environment variable)
+
+deploy/olm-catalog/compliance-operator/$(COURIER_PACKAGE_VERSION):
```

Comment:
Would it make sense to amend the clean target so that it removes this artifact?

Reply:
Not really. Doing that would make our API releases dissappear. Ultimately we do need a policy to clean up older CSVs.